SU-HSDO/suhumsci
# STN-598  private file link color refinement

# READY FOR REVIEW

## Summary
Private file links are currently displayed with two different link colors depending on how they are added: through Text Area as a link or as a Private file and text is displayed. They should be the same link color (active/hover state).
This card makes these links consistent.

Jira Card: [STN-598](https://sparkbox.atlassian.net/browse/STN-598)
Example: https://clas-prod.stanford.edu/academics/student-intranet

## Need Review By (Date)
11/20/20

## Urgency
medium

## Steps to Test
1. npm run test
2. To test, create a Private Page (Add Content > Add Private Page)
- Give the page a title.
- Add a Private Text Area to the components area, add a link in the WYSIWYG area, highlight the link and choose "File Link" from the styles dropdown in the WYSIWYG toolbar.
- Repeat this step for "PDF Link" and "Private Link" in the WYSIWYG link style dropdowns to verify consistency with those links styles as well.
- Additionally, add a "Private Files" component below the "Private Text Area" component you just edited.
- Upload a pdf for testing in that file upload section for the "Private Files" component and give it a description.
- Save the Private Page
3. The link colors and hover states should be consistent.
They should inherit same color value for each pairing/theme (currently secondary color).
![image](https://user-images.githubusercontent.com/9355698/99702932-769c8f00-2a64-11eb-8055-202fc41a79b0.png)



## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
